web/client/components/map/openlayers/Map.jsx

```diff
@@ -303,8 +303,8 @@ class OpenlayersMap extends React.Component {
             }, 0);
         }
 
-        if (this.map && ((this.props.projection !== newProps.projection) || this.haveResolutionsChanged(newProps)) || this.haveRotationChanged(newProps) || this.props.limits !== newProps.limits) {
-            if (this.props.projection !== newProps.projection || this.props.limits !== newProps.limits || this.haveRotationChanged(newProps)) {
+        if (this.map && ((this.props.projection !== newProps.projection) || (this.props.center !== newProps.center) || this.haveResolutionsChanged(newProps)) || this.haveRotationChanged(newProps) || this.props.limits !== newProps.limits) {
+            if (this.props.projection !== newProps.projection || (this.props.center !== newProps.center) || this.props.limits !== newProps.limits || this.haveRotationChanged(newProps)) {
```

Comment:
In this PR #10525 was introduced the [isNearlyEqual](https://github.com/geosolutions-it/MapStore2/pull/10525/files#diff-47404a67952b02809f8241233cc398f3c9b8033c9088e1f9c6e7003ff3471ebdR537-R547) to avoid exact comparation of the center. Shouldn't we use the same approach here?

Reply:
sounds good I guess. i'll update PR